.travis.yml

```diff
@@ -33,6 +33,22 @@ jobs:
       install: pip install moban
       script: .ci/check_moban.sh
 
+    - stage: lint
```

Comment:
You have not used disable_global as requested, or given a reason why it wont work.